content/docs/contribute/contribute@es.md

```diff
@@ -1,5 +1,9 @@
 ---
 $title: Contribuciones
+$order: 0
```

Comment:
For the record, if the English file has these variables in the front matter, any localized files do not need them - they inherit them.  So this change is unnecessary.